katello/katello-change-hostname

```diff
@@ -237,6 +238,12 @@ def hammer_cmd(cmd, exit_codes=[0], message=nil)
   run_cmd("hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} #{cmd}", exit_codes, message)
 end
 
+def get_default_proxy_id
+  output = hammer_cmd("--output json capsule list --search=#{@old_hostname}")
```

Comment:
this should be `--search=name=#{@old_hostname}`, right? otherwise it will find `foo-bar.example.com` when you search `bar.example.com`.